src/main/kotlin/SciviewBridgeUI.kt

```diff
@@ -156,20 +156,20 @@ class SciviewBridgeUI(controlledBridge: SciviewBridge, populateThisContainer: Co
         //
         c.gridx = 1
         INTENSITY_OF_COLORS = SpinnerNumberModel(2400.0, 0.0, 65535.0, 50.0)
-        insertSpinner(INTENSITY_OF_COLORS, { f: Float? -> bridge.intensity.colorIntensity = f!! }, c)
+        insertSpinner(INTENSITY_OF_COLORS, { f: Float -> bridge.intensity.colorIntensity = f }, c)
         c.gridy++
         c.gridx = 0
         insertLabel("When repainting, multiply spots radii with:", c)
         //
         c.gridx = 1
         SPOT_RADIUS_SCALE = SpinnerNumberModel(3.0, 0.0, 50.0, 1.0)
-        insertSpinner(SPOT_RADIUS_SCALE, { f: Float? -> bridge.intensity.spotRadiusScale = f!! }, c)
+        insertSpinner(SPOT_RADIUS_SCALE, { f: Float -> bridge.intensity.spotRadiusScale = f }, c)
         c.gridy++
         INTENSITY_OF_COLORS_BOOST = JCheckBox("Enable enhancing of spot colors when repainting them into the Volume")
         insertCheckBox(INTENSITY_OF_COLORS_BOOST, c)
         c.gridy++
```

Comment:
there's now probably twice added the `c.gridy`.. either it is increased after a new element is added, or it is increased just before adding it.. which suggests which  **one** of the two `c.gridy++` to comment out...